source/client/HandView.py

```diff
@@ -74,9 +81,6 @@ def update(self):
         elif not self.last_hand == self.current_hand:
             self.hand_info = HandManagement.WrapHand(self, self.current_hand, self.hand_info)
         HandManagement.ShowHolding(self, self.hand_info)  # displays hand
-        if self.refresh_flag:  # if needed to rescale card size, then refreshXY again.
```

Comment:
You missed the declaration of refresh_flag at the start of the file if you are purging it entirely